README.md

```diff
@@ -78,64 +81,169 @@ You can run the examples by cloning the repo or copy pasting the below sample in
 ## Example 1 - CBS Authentication
 
 ```js
-const { ConnectionContextBase, CreateConnectionContextBaseParameters, ConnectionConfig } = require("@azure/amqp-common");
-// Using dotenv package is optional and should be used only when .env file is used. 
-// Please take a look at [sample.env](https://github.com/Azure/amqp-common-js/tree/master/sample.env) 
-// file to see how environment variables can be specified.
-import * as dotenv from "dotenv"; 
-dotenv.config();
+const { ConnectionContextBase, ConnectionConfig, CbsResponse }  = require("@azure/amqp-common");
+const dotenv = require("dotenv");
+dotenv.config(); // Optional for loading environment configuration from a .env (config) file
 
-const str = process.env.CONNECTION_STRING || "";
-const path = process.env.ENTITY_PATH;
+export const str = process.env.CONNECTION_STRING || "";
+export const path = process.env.ENTITY_PATH;
+export const connectionConfig = ConnectionConfig.create(str, path);
+const parameters = {
+  config: connectionConfig,
+  connectionProperties: {
+    product: "MSJSClient",
+    userAgent: "/js-amqp-common",
+    version: "0.1.0"
+  }
+};
+export const connectionContext = ConnectionContextBase.create(parameters);
+
+/**
+ * audience The entity token audience in one of the following forms:
+ *
+ * - **ServiceBus**
+ *    - **Sender**
+ *        - `"sb://<yournamespace>.servicebus.windows.net/<queue-name>"`
+ *        - `"sb://<yournamespace>.servicebus.windows.net/<topic-name>"`
+ *
+ *    - **Receiver**
+ *         - `"sb://<yournamespace>.servicebus.windows.net/<queue-name>"`
+ *         - `"sb://<yournamespace>.servicebus.windows.net/<topic-name>"`
+ *
+ *    - **ManagementClient**
+ *         - `"sb://<your-namespace>.servicebus.windows.net/<queue-name>/$management"`.
+ *         - `"sb://<your-namespace>.servicebus.windows.net/<topic-name>/$management"`.
+ * 
+ * - **EventHubs**
+ *     - **Sender**
+ *          - `"sb://<yournamespace>.servicebus.windows.net/<hubName>"`
+ *          - `"sb://<yournamespace>.servicebus.windows.net/<hubName>/Partitions/<partitionId>"`.
+ *
+ *     - **Receiver**
+ *         - `"sb://<your-namespace>.servicebus.windows.net/<event-hub-name>/ConsumerGroups/<consumer-group-name>/Partitions/<partition-id>"`.
+ *
+ *     - **ManagementClient**
+ *         - `"sb://<your-namespace>.servicebus.windows.net/<event-hub-name>/$management"`.
+ */
+export async function authenticate(audience, closeConnection = false) {
```

Comment:
Are there helper functions to assist in creating these audience URL strings?

Reply:
The audience construction happens in the relevant sdks (@azure/event-hubs) and (@azure/service-bus). The intent of the example is to show how things can be done with the lower level library. However, you have made a very good point. We can definitely add some helper methods to create address and audience for a specific entity (EventHub Sender, EventHub Receiver, ServiceBus QueueSender, etc.) from the provided connection string. These methods could be a part of the connection config.